examples/reminders/lib/base/models/reminder/objectbox_reminder_model.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:objectbox/objectbox.dart';
+
+@Entity()
+class ObjectBoxLocalReminderModel {
```

Comment:
Once again the object box classes should not be part of the business layer

Reply:
It sounds like a good option to me. @DDavidPrime what do you think?

Reply:
Yeah, `lib/base/dto/*` or maybe `lib/base/models/dto/*` do sound like a good option.

Reply:
![image](https://github.com/Prime-Holding/rx_bloc/assets/32831935/3a0f2e5f-6af1-4bb8-b9c5-482e5c8c64ff)

Let's go with the lib/base/dtos option